di-ipv-cimit-stub/lambdas/stub-management/src/main/java/uk/gov/di/ipv/core/stubmanagement/service/UserService.java

```diff
@@ -38,11 +38,12 @@ public void addUserCis(String userId, List<UserCisRequest> userCisRequest) {
         userCisRequest.forEach(
                 user -> {
                     Optional<CimitStubItem> cimitStubItem =
-                            getCimitStubItemByCiCode(cimitStubItems, user.getCode());
+                            getCimitStubItemByCiCode(cimitStubItems, user.getCode().toUpperCase());
                     if (cimitStubItem.isEmpty()) {
                         cimitStubItemService.persistCimitStub(
                                 userId,
                                 user.getCode().toUpperCase(),
+                                null,
```

Comment:
Should this be null? This is called when a CI is added with the management API. I think the user who is submitting the request will need to set the issuers on the CI they want to create, and then we pass that through here to be stored.

Reply:
I think we'll also need something in the `else` branch here to set allow the issuer to be updated?